scripts/browse.py

```diff
@@ -74,61 +74,42 @@ def split_text(text, max_length=8192):
         yield "\n".join(current_chunk)
 
 
-def summarize_text(text, is_website=True):
-    if text == "":
+def create_message(chunk, question):
+    return {
+        "role": "user",
+        "content": f"\"\"\"{chunk}\"\"\" Using the above text, please answer the following question: \"{question}\" -- if the question cannot be answered using the text, please summarize the text."
```

Comment:
Users who are not afraid of GPT retaliation should feel free to delete "please" to save tokens :)